dwave/system/utilities.py

```diff
@@ -89,3 +116,114 @@ def get(name, default=False):
     @classproperty
     def hss_solver_config_override(cls):
         return cls.get('hss_solver_config_override')
+
+
+def anneal_schedule_with_offset(
+        anneal_offset: float = 0.0,
+        anneal_schedule: Union[np.typing.ArrayLike, List, None] = None,
+        s: Union[np.typing.ArrayLike, List, None] = None,
+        A: Union[np.typing.ArrayLike, List, None] = None,
+        B: Union[np.typing.ArrayLike, List, None] = None,
+        c: Union[np.typing.ArrayLike, List, None] = None
```

Comment:
do you think both matrix and vectors inputs should be supported? Supporting only one would make the interface simpler.

Reply:
You know we always prefer the Cadillac solution